src/OpenTelemetry.Instrumentation.Hangfire/README.md

```diff
@@ -53,9 +53,6 @@ public class Program
 For an ASP.NET Core application, adding instrumentation is typically done in
 the `ConfigureServices` of your `Startup` class. Refer to [example](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/examples/AspNetCore/Program.cs).
 
-For an ASP.NET application, adding instrumentation is typically done in the
-`Global.asax.cs`. Refer to [example](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/examples/AspNet/Global.asax.cs).
```

Comment:
Instead of removing this link, it could be updated to https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/examples/AspNet/Global.asax.cs. Probably as a relative link.

As I remember this example project was migrated from main repository to the contrib (without updating the link).